packages/shared/tryOnBeforeUnmount/index.md

```diff
@@ -0,0 +1,37 @@
+---
+category: Component
+---
+
+# tryOnUnmounted
```

Comment:
Was this supposed to be renamed too?

Reply:
because tryOnBeforeUnmount use onBeforeUnmount, not onUnmounted